blog/arquillian-core-1.2.0.Final.textile

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+date: 2017-12-20
+author: dipak-pawar
+title: Welcome JUnit Rules for Arquillian
+---
+
+We have released yet another version of Arquillian Core. This time with one new feature.
+
+h3. What's new?
+
+p. <i class="icon-star"></i> **JUnit Rules For Arquillian**
+
+p. If you want to use Arquillian Testing Platform with other JUnit Runner, you can now use JUnit Rules - @ArquillianTestClass@ and @ArquillianTest@ - and happily let Arquillian do the heavy lifting for your tests.
+
+p. To get the similar functionality what the @@RunWith(Arquillian.class)@ is offering you should require both rules i.e. @ArquillianTestClass@ and @ArquillianTest@.
+
+bc(prettify).. @ClassRule
+ public static ArquillianTestClass arquillianTestClass = new ArquillianTestClass();
+
+ @Rule
+ public ArquillianTest arquillianTest = new ArquillianTest();
+
+ @Deployment
+ public static JavaArchive createDeployment() {
+     return ShrinkWrap.create(JavaArchive.class)
+         .addClass(Greeter.class)
+         .addAsManifestResource(EmptyAsset.INSTANCE, "beans.xml");
+ }
+
+ @Inject
+ Greeter greeter;
+
+ @Test
+ public void should_greet_earthlings() throws Exception {
+     String name = "Earthlings";
+     Assert.assertEquals("Hello, " + name, greeter.greet(name));
+ }
+
+p. In order to use any other JUnit Class or Method Rule with @ArquillianClassTest@ or @ArquillianTest@ rule, you have to use it with @RuleChain@ having new methods defined as outer rules. e.g.
+  
+bc(prettify).. @Rule
+public MethodRule testWatchman = MethodRuleChain.outer(new ArquillianTest().around(new TestWatchman());
+
+p. Arquillian is not enforcing user to define both rules as implementations of @MethodRuleChain@ is different for different users. JUnit doesn't provide @MethodRuleChain@ similar to the @RuleChain@, as we don't know your implementation of @MethodRuleChain@, we can't check if your MethodRuleChain has @ArquillianTest@ rule defined.
```

Comment:
I don't really know what you wanted to say here. It's a bit confusing for me. Maybe just worth mentioning that there is no chain for method rules out of the box provided by JUnit (linking to the issue) so we have our own and you can use it if you want

Reply:
I am fine with it.
> There is no chain for method rules out of the box provided by JUnit (https://github.com/junit-team/junit4/issues/383#issuecomment-5056440) so we have our own and you can use it if you want